lib/ui/painting/image_decoder_skia.cc

```diff
@@ -0,0 +1,326 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/lib/ui/painting/image_decoder_skia.h"
+
```

Comment:
Like do we need to retain `#include <algorithm>` here?

Reply:
Done